dbms/src/DataStreams/FilterBlockInputStream.h

```diff
@@ -42,7 +45,25 @@ class FilterBlockInputStream : public IProfilingBlockInputStream
     Block getHeader() const override;
 
 protected:
-    Block readImpl() override;
+    Block readImpl() override
```

Comment:
You can define a mockFilterBlockInputStream and a mock PhysicalFilter to test it

Reply:
The main difficulty is to build `ExpressionActions`